packages/office-ui-fabric-react/src/components/Layer/Layer.styles.ts

```diff
@@ -23,8 +23,8 @@ export const getStyles = (props: ILayerStyleProps): ILayerStyles => {
           zIndex: ZIndexes.Layer,
           top: 0,
           left: 0,
-          width: '100vw',
-          height: '100vh',
+          bottom: 0,
+          right: 0,
```

Comment:
Did you get any indication of why it was the way it was originally? I'd be curious for the reasoning, even if it's just to verify that it was completely incorrect.

Reply:
These styles have been there since the initial conversion from .scss to mergeStyles. I'm not sure who to reach out to about their initial inclusion in the .scss file.

Reply:
Maybe @phkuo or @ThomasMichon would know? This change makes me a little nervous because I remember from when I was on that team working on various issues related to panels' positioning relative to the suite header.

Reply:
You can trace back file history in git to find out who added it originally. Looks like Brian Garland [added it originally](https://github.com/oengusmacinog/office-ui-fabric-react/blob/fa130ecca6509dde9be8ac66227588e3275f32ce/packages/office-ui-fabric-react/src/components/Layer/Layer.styles.ts#L26). Like Elizabeth I'm worried about some subtle regression we'd be introducing.

Reply:
This is fine and the correct fix. The reasoning in the description is exactly right.

Reply:
The only thing we need to be careful about is what element is the parent element of the Layer. Now instead of the Layer always stretching the full width and height of the page, it will only stretch to the dimensions of its parent. Unless we make it position fixed, but that leads to a number of IE11 bugs iirc. David would know more. (There's a reason we banned position:fixed from our repo...)

Reply:
I spoke too soon, it's already position:fixed. Disregard my last comment then.

Reply:
It looks like it [used to be](https://github.com/micahgodbolt/office-ui-fabric-react/blob/78e462bc936350fee2806ee0e70d523e8deee9b5/packages/office-ui-fabric-react/src/components/Layer/Layer.scss) what this PR changes it back to.

And [here's a PR from David](https://github.com/OfficeDev/office-ui-fabric-react/pull/659) that changed it to `100vh/vw`.

Maybe @dzearing remembers?

![image](https://user-images.githubusercontent.com/26070760/60302766-9d937780-98e9-11e9-9672-4447d7b7267f.png)

Reply:
Nice detective work.
It looks like even if you do weird things with <html> or <body>, fixed will ALWAYS be relative to the viewport, so I don't see how bottom/right:0 would ever break.

Reply:
vh and vw were proven to not work on Safari. My bad if I introduced that.